app/form_models/publishers/job_application/tag_form.rb

```diff
@@ -1,8 +1,30 @@
 class Publishers::JobApplication::TagForm
   include ActiveModel::Model
   include ActiveModel::Validations
+  include ActiveModel::Attributes
 
-  attr_accessor :job_applications
+  attribute :status, :string
+  attr_accessor :job_applications, :origin
 
   validates_length_of :job_applications, minimum: 1
+  validates_presence_of :status, on: :update_tag
```

Comment:
I don't understand why we would ever not validate the presence of status? We don't currently (which is a slight oversight as we always provide it) but only doing it 'sometimes' feels very odd